pkg/kubectl/resource_printer.go

```diff
@@ -60,6 +60,9 @@ const (
 	loadBalancerWidth = 16
 )
 
+// This indicates that a particular StorageClass nominates itself as the system default.
+const betaAnnotation = "storageclass.beta.kubernetes.io/is-default-class"
```

Comment:
We used the name `default` as the default serviceaccount and that has worked reasonably well for us.  It ensures nothing collides (can't have two defaults) and avoids special markings like this.